src/pseudopeople/interface.py

```diff
@@ -155,10 +155,10 @@ def generate_decennial_census(
     :param seed: An integer seed for randomness.
     :param config: An optional override to the default configuration. Can be a path
         to a configuration YAML file or a dictionary.
-    :param year: The year (format YYYY) to include in the dataset. Must be a decennial
-        year (e.g. 2020, 2030, 2040). Will raise a ValueError if there is no data for
-        this year. If None is provided, data for all years are
-        included in the dataset.
+    :param year: The year in which the census was conducted (format YYYY, e.g., 2030). Must be a decennial
```

Comment:
This past tense feels a little odd to me given that this is a parameter the user is choosing. How would you feel about something like this?

```suggestion
    :param year: The year to conduct the simulated census (format YYYY, e.g., 2030). Must be a decennial
```

Or, one more idea: "The year to conduct the census in the simulated population"